sdk/logz/Microsoft.Azure.Management.Logz/src/Microsoft.Azure.Management.Logz.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.Logz</PackageId>
+    <Description>Microsoft Azure Management Logz library</Description>
+    <Version>1.0.0</Version>
```

Comment:
First version should be 1.0.0-beta.1

Reply:
Thank you for the feedback
Changed the version